flink-dist/src/main/resources/META-INF/NOTICE

```diff
@@ -12,7 +12,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.twitter:chill_2.11:0.7.6
 - com.typesafe:config:1.3.0
 - com.typesafe:ssl-config-core_2.11:0.3.7
-- com.typesafe.akka:akka-actor_2.11:2.5.1
+- com.typesafe.akka:akka-actor_2.11:2.5.21
```

Comment:
Isn't
```
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.2 in the shaded jar.
```
missing from this file?
(my mvn output is from 1.11 and a few days old 🤦 )

Reply:
I have
```
[INFO] Including org.lz4:lz4-java:jar:1.5.0 in the shaded jar.
```
in my output, but this file lists version 1.6.0. Is this a 1.11 vs 1.12 issue?

Reply:
lz4-java was bumped in https://github.com/apache/flink/commit/889705c7ab580737f0650f39dcd3712ec19c99e9 and is available in both master and release-1.11 . Was your branch possibly outdated?

Reply:
reactive-streams  is at the very bottom ;)

Reply:
Ah, I didn't know that you have to click this "expand" button multiple times. My bad.

Reply:
> lz4-java was bumped in 889705c and is available in both master and release-1.11 . Was your branch possibly outdated?

Yes, I was outdated